src/abstractdataframe/io.jl

```diff
@@ -298,31 +322,37 @@ function Base.show(io::IO, mime::MIME"text/html", gd::GroupedDataFrame)
         nrows = size(gd[1], 1)
         rows = nrows > 1 ? "rows" : "row"
 
-        identified_groups = [html_escape(string(col, " = ",
-                                                repr(first(gd[1][!, col]))))
+        identified_groups = [string(col, " = ", repr(first(gd[1][!, col])))
                              for col in gd.cols]
 
-        write(io, "<p><i>First Group ($nrows $rows): ")
-        join(io, identified_groups, ", ")
-        write(io, "</i></p>")
-        show(io, mime, gd[1], summary=false)
+        title = "First Group ($nrows $rows): " * join(identified_groups, ", ")
+        _show(io, mime, gd[1], title=title)
     end
     if N > 1
         nrows = size(gd[N], 1)
         rows = nrows > 1 ? "rows" : "row"
 
-        identified_groups = [html_escape(string(col, " = ",
-                                                repr(first(gd[N][!, col]))))
+        identified_groups = [string(col, " = ", repr(first(gd[N][!, col])))
                              for col in gd.cols]
 
         write(io, "<p>&vellip;</p>")
-        write(io, "<p><i>Last Group ($nrows $rows): ")
-        join(io, identified_groups, ", ")
-        write(io, "</i></p>")
-        show(io, mime, gd[N], summary=false)
+        title = "Last Group ($nrows $rows): " * join(identified_groups, ", ")
+        _show(io, mime, gd[N], title=title)
     end
 end
 
+# Internal function to verify the keywords in show functions using the HTML
+# backend.
+function _verify_kwargs_for_html(; kwargs...)
+    haskey(kwargs, :rowid) &&
+        throw(ArgumentError("The keyword `rowid` is reserved and must not be used."))
+
+    haskey(kwargs, :title) &&
+        throw(ArgumentError("Use the keyword `top_left_str` instead of `title` to change the label above the data frame."))
```

Comment:
```suggestion
        throw(ArgumentError("Use the `top_left_str` keyword argument instead of `title`" *
                            "to change the label above the data frame."))
```